integrations/viewing-data-in-an-integration.md

```diff
@@ -1,5 +1,24 @@
 # Viewing Data in an Integration
 
-We're hard at work on a new feature that will show you a preview of the data in your connected integrations, but it's not quite ready yet.
+## Base Data Artifacts
+These are the data artifacts that exist in the integration that wasn't loaded there via the Aqueduct `.load` function.
+### Aqueduct UI
+These can be found in the integration details page in the UI. You can bring up the integration details page by:
+1. Navigate to the Integrations page via the tab labeled "Integrations" in the left side pane.
```

Comment:
Nit -- I don't think we need to tell people that the button labeled integrations takes them to the integrations page. 🙂